tests/Fixtures/DummyObjectWithOnlyPrivateProperty.php

```diff
@@ -15,5 +15,4 @@
 
 class DummyObjectWithOnlyPrivateProperty
 {
-    private $foo;
```

Comment:
These removals look faulty to me.

Reply:
Ok but there is this PHPStan error : Property ApiPlatform\Core\Tests\Fixtures\DummyObjectWithOnlyPrivateProperty::$foo is unused.